tools/src/main/scala/scala/scalanative/build/Config.scala

```diff
@@ -196,6 +196,12 @@ sealed trait Config {
           .getOrElse(compilerConfig.mode.isInstanceOf[Mode.Release])
       case _ => false
     }
+
+  private[scalanative] def isCPPRuntimeEnabled: Boolean =
+    targetsWindows || compilerConfig.useCPPRuntime
+
+  private[scalanative] def usingCPPExceptions: Boolean =
+    targetsWindows
```

Comment:
Maybe instead let's call it `requireCppRuntime` so that we can actually detect if the Cxx is needed, we can then also deduplicate some detections. 

```suggestion
    private[scalanative] lazy val requiresCppRuntime: Boolean =
    usingCPPExceptions || compilerConfig.useCPPRuntime

  private[scalanative] def usingCPPExceptions: Boolean =
    targetsWindows
```